cmd/sovereignnode/incomingHeader/incomingHeaderProcessor_test.go

```diff
@@ -654,14 +779,83 @@ func TestIncomingHeaderHandler_AddHeader(t *testing.T) {
 	wasOutGoingOpConfirmed := false
 	args.OutGoingOperationsPool = &sovTests.OutGoingOperationsPoolMock{
 		ConfirmOperationCalled: func(hashOfHashes []byte, hash []byte) error {
-			require.Equal(t, topic3[0], hashOfHashes)
-			require.Equal(t, topic3[1], hash)
+			require.Equal(t, topic3[0], []byte(topicIDConfirmedOutGoingOperation))
+			require.Equal(t, topic3[1], hashOfHashes)
+			require.Equal(t, topic3[2], hash)
 
 			wasOutGoingOpConfirmed = true
 			return nil
 		},
 	}
 
+	tcCalled := -1
```

Comment:
Please use more descriptive names for these counters like `checkValidityCt` and below